.nvmrc

```diff
@@ -1 +1 @@
-10.16
+12.16.3
```

Comment:
You mentioned needing node > 12. Is this the new minimum?

Reply:
The code uses a few JavaScript features that are only available in Node v12. I'm not sure how many are in Node >v12.0.0, but since the production code is running on Node v12.6.3 (due to the version of nodejs-mobile we use), I thought it best to target this specific version for development and testing.